Library/Homebrew/api.rb

```diff
@@ -15,12 +15,10 @@ module API
 
     extend Cachable
 
-    module_function
-
     HOMEBREW_CACHE_API = (HOMEBREW_CACHE/"api").freeze
 
     sig { params(endpoint: String).returns(Hash) }
-    def fetch(endpoint)
+    def self.fetch(endpoint)
```

Comment:
Note that this is a style deviation from the files under `Library/Homebrew/api/`, which use the `class << self` syntax to define class methods. I prefer the style here, as it makes it obvious at the `def` where the method is being defined, but I'm equally happy to match the preferred style.